src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/CustomiseHearingBundleConfirmationTest.java

```diff
@@ -60,6 +69,39 @@ public void should_return_confirmation() {
         );
     }
 
+    @Test
+    public void should_return_notification_failed_confirmation() {
+
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(asylumCase);
+
+        when(callback.getEvent()).thenReturn(Event.CUSTOMISE_HEARING_BUNDLE);
+        when(asylumCase.read(AsylumCaseFieldDefinition.HOME_OFFICE_HEARING_BUNDLE_READY_INSTRUCT_STATUS, String.class))
+            .thenReturn(Optional.of("FAIL"));
+
+        PostSubmitCallbackResponse callbackResponse =
+            customiseHearingBundleConfirmation.handle(callback);
+
+        Assert.assertNotNull(callbackResponse);
+        Assertions.assertThat(callbackResponse.getConfirmationHeader()).isNotPresent();
+        Assert.assertTrue(callbackResponse.getConfirmationBody().isPresent());
```

Comment:
using assertThat from two different libraries. Could we be consistent here? may be use just the assertj library?

Reply:
Corrected Assert statements